src/wp-includes/kses.php

```diff
@@ -250,9 +253,11 @@
 		'p'          => array(
 			'align' => true,
 		),
+		'picture'    => array(),
 		'pre'        => array(
 			'width' => true,
 		),
+		'progress'   => array(),
```

Comment:
There are a [couple attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/progress#attributes) that this should allow.

Reply:
Also, if `progress` is added, then why not `meter`?

Reply:
Related: https://github.com/WordPress/gutenberg/pull/68652